### DISCUSSION TITLE
Suggest : update.bat

### DISCUSSION THREAD

**Discussion by yo1nkers** [vladmandic/sdnext#2393] [Ideas]
Hi! I'd like to suggest an update.bat file, a file tasked with the simple job of updating the branch. My girlfriend loves to use this, and I help her wherever I can as she's usually at mine, but a lot of the time she isn't and it would help greatly if there were to be a bat file for updating it to the latest branch.

I can see this being useful for others that do not use git pull.

**Comment by Woisek** [comment]
Don't you have a webui-user.bat file for using command lines already? 🤔
Otherwise, just a create a .bat file that is calling webui.bat and put a git pull in front of it.

**Comment by Tillerz** [comment]
What Woisek said. Just create a .bat file yourself. SD won't delete it.

webui-user.bat
```batch
@echo off

call webui.bat --upgrade
```

**Reply by Woisek** [reply]
<strike>But it needs the git pull in front of the call, otherwise it don't do updates. 😉</strike>

I didn't notice the --upgrade

**Reply by brknsoul** [reply]
It doesn't. The --upgrade command references git pull already.

**Comment by vladmandic** [comment]
`git pull` is for users that want to do everything manually. its is NOT required by any means.
stardard way of updating is `webui --upgrade` and thats it.
and definitely don't do git pull and --upgrade together.

**Comment by ETA-22** [comment]
I just run with —upgrade as a default in the command line, so every time I start, everything gets updated (for better or worse).

**Reply by vladmandic** [reply]
ok, maybe not **every** time - you probably want to know when a major upgrade is happening?

**Comment by cibernicola** [comment]
I did this: https://github.com/cibernicola/bat-launcher-menu/blob/main/launcher
